public/v1/components/schemas.yaml

```diff
@@ -1276,8 +1280,6 @@ components:
       title: TlsCertificate
       required:
         - authorized
-        - createdAt
-        - expiresAt
```

Comment:
These don't seem to be optional in the node.js definitions. Are they?

Reply:
I didn't find such a measurement, but it is possible according to the:

https://github.com/jsdelivr/globalping-probe/blob/aa723923ad42c3ec422c837441dc8f4d5d5e711d/src/command/http-command.ts#L407-L410

Reply:
Based on node.js docs the values are always there. It seems the condition was [test-related](https://github.com/jsdelivr/globalping-probe/commit/80082a131622a68f9f0090dba8bc74de67985328), but at the moment, even the tests seem fine without it. So let's keep this required here and possibly remove the condition on the probe side as well.